docs/blogs/GitSupport.md

```diff
@@ -383,7 +383,7 @@ secrets:
 
 note: this secret cannot currently be stored in a vault - see future work.
```

Comment:
Please can you clarify line 384 and c2.  It say's secrets cannot be stored in a vault here but it talks about adding secret to vaults in line 404